images/epg-help.xpm

```diff
@@ -1,248 +1,257 @@
 /* XPM */
-static char * epg_help_xpm[] = {
-"24 24 221 2",
+static char * help_xpm[] = {
```

Comment:
Should this be called epg_help_xmp intead of help_xmp? (Does it not make a difference?)